stable/xray/templates/xray-statefulset.yaml

```diff
@@ -189,13 +187,14 @@ spec:
           valueFrom:
             secretKeyRef:
         {{- if .Values.database.secrets.url }}
-              name: {{ .Values.database.secrets.url.name }}
-              key: {{ .Values.database.secrets.url.key }}
+              name: {{ tpl .Values.database.secrets.url.name . }}
+              key: {{ tpl .Values.database.secrets.url.key . }}
         {{- else if .Values.database.url }}
               name: {{ template "xray.fullname" . }}-database-creds
               key: db-url
         {{- end }}
       {{- end }}
+      {{- if .Values.common.rabbitmq.connectionConfigFromEnvironment }}
```

Comment:
@chukka why do we need this? Not sure I understand the use case for setting this to false

Reply:
@danielezer  refer here for more details - https://github.com/jfrog/charts/pull/924/files/2c1886c693cb19b1b2c0fb5c269620ba676eaa21#diff-153bb5de1ec821f30fb4d9703c126d11R284